src/CommandLine.hs

```diff
@@ -42,21 +42,31 @@ defaultCommandLineArgs = CommandLineArgs
 
 -- | These arguments are used by the `test` target.
 data TestArgs = TestArgs
-    { testOnly     :: Maybe String
+    { testConfigs  :: [String]
+    , testJUnit    :: Maybe FilePath
+    , testOnly     :: Maybe String
+    , testOnlyPerf :: Bool
     , testSkipPerf :: Bool
+    , testSpeed    :: TestSpeed
     , testSummary  :: Maybe FilePath
-    , testJUnit    :: Maybe FilePath
-    , testConfigs  :: [String] }
+    , testThreads  :: Maybe String
+    , testVerbosity:: Maybe String
```

Comment:
Hadrian (or Shake) already has `-jN` option, which is enough to control the speed of running testsuite. We don't need the `testThreads` here.

Reply:
@sighingnow Indeed! Well spotted.